cmd/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	//"github.com/go-playground/validator/v10"
+	"github.com/joho/godotenv"
+	d "github.com/kmakasheva/todo-list-project/db"
+	"net/http"
+	"os"
+)
+
+func main() {
+	d.CreateDB()
+
+	r := SetupRouter()
+
+	err := godotenv.Load()
+	if err != nil {
+		fmt.Println("Error loading .env file")
```

Comment:
1. Теряешь текст ошибки
2. Программа выходит с кодом 0, что не очень

```suggestion
		log.Fatalf("Error loading .env file: %v", err)
```